package.json

```diff
@@ -124,6 +130,7 @@
     "sync-monorepo-packages": "0.3.4",
     "through2": "4.0.2",
     "type-fest": "2.8.0",
+    "typescript": "4.6.0-dev.20220210",
```

Comment:
yeah, last time I checked, "it doesn't work".  I'll try again though with 4.5.x

Reply:
ok. should we add a comment in here like:

```
"devDependencies": {
  ...,
  "//": {
    "typescript": "4.5.x doesn't work because reasons"
  }
}
```